windows/src/developer/TIKE/main/UfrmMain.pas

```diff
@@ -530,15 +529,6 @@ procedure TfrmKeymanDeveloper.FormCreate(Sender: TObject);
 
   frmHelp := TfrmHelp.Create(Application);
 
-  KL.Log('GetDebugManager start');

```

Comment:
The debug logging only needs to be started in keyman.exe for now. (if we refactor debugging further later, this may move to the Keyman COM API)